umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -6,11 +6,60 @@
   [cmr.umm-spec.date-util :as date-util]
   [cmr.umm-spec.umm-spec-core :as spec-core]))
 
+(def DataCenterURL
+  "Data center URL's content type."
+  "DataCenterURL")
+
+(def HomePage
+  "Data center URL's type."
+  "HOME PAGE")
+
 (defn field-update-functions
   "Partial update functions for handling specific update cases"
   [umm]
   {[:Instruments] (partial util/update-in-each umm [:Platforms])})
 
+(defn- get-home-page-url
+  "Get data center home page url from a list of related urls."
+  [related-urls]
+  (when (seq related-urls)
+    (first (for [{:keys [URLContentType Type URL]} related-urls
+                 :when (and (= DataCenterURL URLContentType)
+                            (= HomePage Type))]
+             {:URLContentType DataCenterURL
+              :Type HomePage
+              :URL URL})))) 
+
+(defn- home-page-url?
+  "Check if a related url is a data center home page url."
+  [related-url]
+  (let [{:keys [URLContentType Type URL]} related-url]
+    (if (and (= DataCenterURL URLContentType)
+             (= HomePage Type))
+      true
+      false)))
+
+(defn- home-page-url-update
+  "Apply special homepageurl update to data-center using the update-value.
```

Comment:
Add dashes between all words in the docstring and the bindings in the code to make it easier to read:
e.g.
homepageurl -> home-page-url
update-value-non-contactinfo-part -> update-value-non-contact-info-part

Reply:
Done.